attributes/default.rb

```diff
@@ -18,5 +18,5 @@
 # limitations under the License.
 #
 
-default['gluster']['version'] = '3.4'
+default['gluster']['version'] = '3.6'
```

Comment:
Either way, this can be a new pull request if we would like to push to 3.7, I don't want to get held back with testing 3.7 when it's not the focus of this PR.